### PR TITLE
Fix #4897 - report css...

### LINKED ISSUE #4897: PDF export of general report does not (always?) show inheritance patterns for OMIM terms
Thank you for taking the time to contribute an issue! 

**Describe the bug**

In the last release we removed display of inheritance patterns for ORPHA terms and introduced variant page formatting/coloring of OMIM inheritance patterns. Apparently these don't render well on pdf/paper in all circumstances. Could be the specific inheritance pattern formatting, or something from the new import of the general css.

image

**To Reproduce**
1. Pin or mark a variant in a gene with a disorder
2. Get a general report, note the inheritance patterns
3. Make a pdf, and notice some/all inheritance patterns disappear

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by dnil** [comment]
So the primary issue appears to be that wkhtmltopdf does not find the local static URL path. An option is to use an option external=True for the url_from call. I still had messy pdfs locally when trying this, although not the same as the prod issue. I still have that with old reverts locally, so likely my local pdf generation env was a bit off as well. This seems to do the trick on stage, but we could retest with the full css as well.

File: scout/server/blueprints/cases/templates/cases/case_report.html [hunk 1]
```diff
@@ -1099,7 +1099,9 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
               <td>
                 <ul>
                 {% for disease_term in gene.disease_terms %}
-                   <li class="d-flex align-items-baseline"><span class="badge bg-secondary m-1">{{ disease_term._id}}</span>&nbsp;<span>{{ disease_term.description }}&nbsp;{{ disease_term.inheritance }}</span></li>
+                   <li class="d-flex align-items-baseline"><span class="badge bg-secondary m-1">{{ disease_term._id}}</span>&nbsp;<span>{{ disease_term.description }}&nbsp;{% if disease_term.source != 'ORPHA' and disease_term.inheritance %}
```
Comment (dnil) [inline scout/server/blueprints/cases/templates/cases/case_report.html:1102]: Also noticed that the SVs have a cut-and-pasted version of the same table. 😊

File: scout/server/blueprints/cases/templates/cases/case_report.html [hunk 2]
```diff
@@ -1140,7 +1142,7 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
     {{ variant_phenotypes(variant) }}
   </div>
   <div class="card-body">
-    <table id="panel-table" class="table table-sm table-bordered" style="background-color: transparent; margin-left:-.3rem;>
+    <table id="panel-table" class="table table-sm table-bordered" style="background-color: transparent; margin-left:-.3rem;">
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/templates/cases/case_report.html:1145]: 💯

**Review by northwestwitch — APPROVED** [review]
Tested on stage and seems to work fine, so I think this PR is good enough to merge. If the problem shows up again we can attempt the fix you suggest (the url_for thing), I think

**Pull request merged**